docs/content/02-tutorials/07-cost-tab.md

```diff
@@ -0,0 +1,56 @@
+
++++
+title = "f. Cost Tab 📈"
+weight = 27
++++
+
+This tutorial tells you how to view cluster spending costs and created budgets for clusters
+
+## Setup
+
+1. Cost Allocation Tags allow us to track spending of parallelclusters from parallelcluster specific tags, so first we must activate those tags.
+2. Press the "Activate Tags" button to activate Cost Allocation Tags
+![Cost Tags](cost-tab/Activate-Tags-Button.png)
+
+3. If cost allocation tags were already activated, then the Activate Tags Button should look "grayed out"
+![Cost Tags](cost-tab/Activated_Button.png)
+
+## Filter Data by dates
+
+Now you can check spending costs by cluster based on range of specified dates
```

Comment:
> Now you can check spending costs by cluster based on range of specified dates

Should become "Now you can check spending costs by cluster based and filter by time range."

Reply:
Resolved